packages/dds/map/src/mapKernel.ts

```diff
@@ -585,12 +652,12 @@ export class MapKernel {
                     this.clearCore(local);
                 },
                 submit: (op: IMapClearOperation, localOpMetadata: unknown) => {
-                    // We don't reuse the metadata but send a new one on each submit.
-                    this.submitMapClearMessage(op);
+                    // We don't reuse the metadata pendingMessageId but send a new one on each submit.
+                    this.submitMapClearMessage(op, (localOpMetadata as IMapClearLocalOpMetadata).previousMap);
```

Comment:
why cast and pull out part here, rather than just passing the whole thing?

Reply:
this goes to my above comment about local metadata typing. i'd rather it be simplier, just pass the whole thing around, and more strongly typed

Reply:
submitMapClearMessage is also called by `clear`, which has no prior localOpMetadata.